aggregated-dependency-score.go

```diff
@@ -60,7 +60,7 @@ type DependencyResolver interface {
 	GetDirectDependencies(ctx context.Context, p Package) ([]Package, error)
 }
 
-func (eval *trustwhorthinessEvaluator) evaluate(ctx context.Context, p Package) (float64, error) {
+func (eval *trustwhorthinessEvaluator) evaluate(ctx context.Context, p Package, ancestors map[string]struct{}) (float64, error) {
```

Comment:
nit: "eval" feels like a misnomer as it feels like a typical evaluate function

Reply:
Good point, fixed in [538059f](https://github.com/DataDog/aggregated-dependency-score/pull/10/commits/538059ff85ffaf837bf6bef874a03f69ca0e1c7e)